bin/restful-react-import.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+const program = require("commander");
+const { join } = require("path");
+const importOpenApi = require("../dist/import-open-api");
```

Comment:
🤔

Reply:
This file is in `bin` and a part of the restful-react build, so he take the generated `.js` file from `dist` ;)

Reply:
This is quite confusing. Can we think of a clearer way to express this in code?